deploy/1_deploy.js

```diff
@@ -72,7 +72,7 @@ module.exports = async ({ getUnnamedAccounts, deployments }) => {
     log(`Deployed PrepaymentDepository (OEV Relay) at ${prepaymentDepository.address}`);
   }
 
-  if (chainsSupportedByChainApi.includes(network.name)) {
+  if (chainsSupportedByChainApi.includes(network.name) || network.name === 'ethereum-sepolia-testnet') {
```

Comment:
The alternative is to keep a list of chains per contract but it feels like that will be more error-prone (we will copy-paste an old version of the list to the new contract, etc.)

Reply:
I think this hardcoding is fine given `RequesterAuthorizerWithErc721` isn't meant to be deployed on a large number of chains (as opposed to say something like `AirnodeRrpV0`). My only other thought was that there could be another export below here:
https://github.com/api3dao/airnode-protocol-v1/blob/f22a312b415bbebb287150460e3575a33193f5a3/src/supported-chains.js#L16

named something like `additionalRequesterAuthorizerWithErc721Chains` that complements `chainsSupportedByChainApi`, resulting in the conditional:
```ts
if (chainsSupportedByChainApi.includes(network.name) || additionalRequesterAuthorizerWithErc721Chains.includes(network.name))
```